tests/unit/stream_alert_cli/test_config.py

```diff
@@ -72,8 +36,149 @@ def test_load_config():
     with mock_open(global_file, global_contents):
         with mock_open(lambda_file, lambda_contents):
             # mock os call
-
             # test valid and invalid clusters
-
             config = CLIConfig()
             assert_equal(config['global']['account']['prefix'], 'unit-testing')
+
+
+@patch('logging.Logger.error')
+@patch('stream_alert_cli.config.CLIConfig.write')
+def test_toggle_metric(write_mock, log_mock):
+    """CLI - Metric toggling"""
+    config_data = basic_streamalert_config()
+
+    global_file = 'conf/global.json'
+    global_contents = json.dumps(config_data['global'], indent=2)
+
+    lambda_file = 'conf/lambda.json'
+    lambda_contents = json.dumps(config_data['lambda'], indent=2)
+
+    with mock_open(global_file, global_contents):
+        with mock_open(lambda_file, lambda_contents):
```

Comment:
A Python3 feature which has also been backported to 2.7:

```python
with mock_open(global_file, global_contents), mock_open(lambda_file, lambda_contents):
   # do stuff
```

Reply:
Here and throughout (this is in every test you added)

Reply:
Nice i hadn't realized that was backported. Was just copy pasta from some tests someone else added